### DISCUSSION TITLE
Ory Open Source Support Survey: We want your input!

### DISCUSSION THREAD

**Discussion by vinckr** [ory/kratos#3919] [Ideas]
Are you running Ory Hydra, Kratos, Keto, or Oathkeeper in production?
Let us know your support needs!
Complete our 5-minute survey to influence the development of new support options for self-hosted Ory:
[Link to the survey](https://share-eu1.hsforms.com/15DiCnJpcRuijnpAdnDhxxwextgn)

**Comment by IngwiePhoenix** [comment]
Interesting, will take it!
I am [currently stuck](https://github.com/ory/kratos/discussions/3965) with the setup but I am looking forward to learning more; and help out where i can ^^

**Reply by IngwiePhoenix** [reply]
Side note: Please don't immediately redirect me to a meetings page right after the survey.
That was weird... I am happy to give feedback, but I didn't want to sign up for a meeting.

**Reply by vinckr** [reply]
@IngwiePhoenix 

Gotcha, thanks for the feedback on both poins :-)
